.storybook/preview-head.html

```diff
@@ -0,0 +1,2 @@
+<link href='https://fonts.googleapis.com/css?family=Roboto:400,300italic,400italic,300,700,700italic|Oswald:300,400,700' rel='stylesheet' type='text/css'>
+<link rel="stylesheet" media="screen" href="${SERVER_URL}/assets/darkswarm/all.css" />
```

Comment:
This doesn't work?
```suggestion
<link rel="stylesheet" media="screen" href="/assets/darkswarm/all.css" />
```

Reply:
When we re-design the backoffice, it would be good to make it compatible with the frontend styles, maybe even build on top of them. Then we can include them both here but normal shoppers will need only the frontend styles.

Reply:
> This doesn't work?

Actually not, because it's then served by the Storybook server (ie. `http://localhost:52533/assets/darkswarm/all.css`). We need to specify the Rails server to get the right assets. 

> When we re-design the backoffice, it would be good to make it compatible with the frontend styles, maybe even build on top of them. Then we can include them both here but normal shoppers will need only the frontend styles.

That would be a good idea. I've tested (a little bit) to share CSS between back and front office (for the unit price tooltip) and it was a bit painful. But maybe this is not awful as I saw. Worth a try!

Reply:
Interesting. I expected the path to be relative to the host from which it's loaded but Storybook works differently here. I'm not sure if it's the fact that the element is injected via JS or maybe it's Storybook injecting the code into the page instead of just using an iframe. :shrug: 

Anyway. The current solution doesn't seem to work either. The variable doesn't get replaced in time and my browser tries to load the raw URL. I'll add a commit with another solution.

Reply:
Thanks.